CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Improvements:
 - Add option to disable "Not all open documents were saved" popup. [#2889](https://github.com/microsoft/vscode-cmake-tools/issues/2889)
 - Allow overriding of CMakePresets cache variables and arguments. [#1836](https://github.com/microsoft/vscode-cmake-tools/issues/1836)
 - Improve advanced status bar options configuration by adding an `inherit` option to the `statusBarVisibility` setting and by adding the `inheritDefault` setting. Look at the CMake Options Visibility Configuration docs for more information. [#3452](https://github.com/microsoft/vscode-cmake-tools/issues/3452)
+- Add option to disable kit scan by default when a kit isn't selected. [#1461](https://github.com/microsoft/vscode-cmake-tools/issues/1461)
```

Comment:
This is where it would normally go, but unfortunately you had the bad luck of implementing this right during a release cycle 😂

So this will actually go in a 1.18 new section, which we can create after we release next week. 

Everything looks good other than this 👍